sale_stock_picking_note/models/sale_order.py

```diff
@@ -0,0 +1,30 @@
+# Copyright 2018 Tecnativa - Carlos Dauden
+# Copyright 2021 Tecnativa - Víctor Martínez
+# Copyright 2023 Moduon Team S.L.
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class SaleOrder(models.Model):
+    _inherit = "sale.order"
+
+    picking_note = fields.Html(
+        string="Picking Internal Note",
+        compute="_compute_picking_notes",
+        store=True,
+        readonly=False,
+    )
+    picking_customer_note = fields.Text(
+        string="Picking Customer Comments",
+        compute="_compute_picking_notes",
+        store=True,
+        readonly=False,
+    )
+
+    @api.depends("partner_id.picking_note", "partner_id.picking_customer_note")
```

Comment:
suggestion: this isn't a very good idea as you'll squash a lot of custom notes without noticing. Just depend on `partner_id`.